src/hotspot/share/runtime/javaThread.hpp

```diff
@@ -784,10 +784,10 @@ class JavaThread: public Thread {
   void set_callee_target  (Method* x)            { _callee_target   = x; }
 
   // Oop results of vm runtime calls
-  oop  vm_result() const                         { return _vm_result; }
-  void set_vm_result  (oop x)                    { _vm_result   = x; }
+  oop  vm_result_oop() const                     { return _vm_result_oop; }
+  void set_vm_result_oop(oop x)                  { _vm_result_oop   = x; }
```

Comment:
```suggestion
  void set_vm_result_oop(oop x)                  { _vm_result_oop = x; }
```

Reply:
I don't think Robert fixed the space here.